pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "nwp-demo"
+# Keep only Major and Minor version in this configuration,
+# the patch version is calculated when releasing the library
+version = "1.0"
+description = "Demo scripts and notebooks for reading and processing forecast data using Polytope."
+authors = ["MeteoSwiss"]
+package-mode = false
+
+[[tool.poetry.source]]
+name = "pypi-mch"
+url = "https://service.meteoswiss.ch/nexus/repository/python-all/simple"
+
+[tool.poetry.dependencies]
+python = "~3.11"
+meteodata-lab = { git = "https://github.com/MeteoSwiss/meteodata-lab.git", branch = "main", extras = ["all"] }
+pyfdb = "^0.0.3"
```

Comment:
What version of pyfdb is installed?

Reply:
We have to use pyfdb 0.0.3 due to the request expansion issue. Maybe there's a newer pyfdb version that no longer has this issue. However, I'm asking because I don't believe that you are instructing poetry to install any version at all (pyfdb is not available on pypi)

Reply:
That was probably from meteodata-lab but, since need to push to pypi and direct dependencies are not allowed, this will no longer be the case I believe